index.html

```diff
@@ -893,13 +893,17 @@ <h4><code>Timing-Allow-Origin</code> Response Header</h4>
 </li>
 <li>
 <p>If the <a>Timing-Allow-Origin</a> header value list contains a
-case-sensitive match for the value of the <code><a data-cite=
-"RFC6454#section-4" data-lt="http-origin">origin</a></code> of the
-<a>current document</a>, or a wildcard ("<code>*</code>"), return
-<code>pass</code>.</p>
+value which is byte-for-byte identical to the <a
+data-cite="HTML#ascii-serialisation-of-an-origin">serialisation</a>
```

Comment:
serialization*, unless you really use en-GB. (I realize it's confusing that the fragment does use en-GB, but that's hard to change other than keeping two long fragments around forever.)

Reply:
I really don't. The laptop I switched to recently keeps trying to force me into GB spelling, which is probably the cause here.